ctk/common/src/main/kotlin/org/codice/compliance/verification/binding/PostBindingVerifier.kt

```diff
@@ -35,7 +35,6 @@ import kotlin.test.assertNotNull
 class PostBindingVerifier(httpResponse: Response) : BindingVerifier(httpResponse) {
     /** Verify the response for a post binding */
     override fun decodeAndVerify(): Node {
-        verifyHttpStatusCode(httpResponse.statusCode)
```

Comment:
Taking this out of the verify method because it will always be called directly from the tests.

Reply:
✏️ I think we should delete it from the tests and leave it here. For me, it makes sense for the Binding verifier to call it versus each test. If the issue is the `UnsupportedOperationException` couldn't we catch that and throw a `SAMLComplianceException`?

Reply:
A problem with that approach is that if they actually do have a binding we don't support, we may catch that error and throw a misleading "Incorrect http status code" `SAMLComplianceException`.